src/nvim/buffer.h

```diff
@@ -15,9 +15,11 @@ enum getf_values {
 
 // Values for buflist_new() flags
 enum bln_values {
-  BLN_CURBUF = 1, // May re-use curbuf for new buffer
-  BLN_LISTED = 2, // Put new buffer in buffer list
-  BLN_DUMMY  = 4, // Allocating dummy buffer
+  BLN_CURBUF = 1,   // May re-use curbuf for new buffer
+  BLN_LISTED = 2,   // Put new buffer in buffer list
+  BLN_DUMMY  = 4,   // Allocating dummy buffer
+  // TODO(mhinz): merge patch that introduces BLN_NEW
```

Comment:
Ah, okay. It's not important for this PR, though. Just thought I leave a note so people don't wonder we jumped from 4 to 16.